composer.json

```diff
@@ -20,7 +20,8 @@
     "require": {
         "php": ">=8.2",
         "ext-curl": "*",
-        "ext-json": "*"
+        "ext-json": "*",
+        "symfony/deprecation-contracts": "^3.5"
```

Comment:
forgot to add this @slunak

Reply:
I actually would like to avoid any dependencies. I would be ok to use point 3 above.

Reply:
Maybe I come back later here, feel free to take over, I am a bit busy at the moment. sorry

Reply:
@OskarStark I have removed the dependency. I wonder, if `trigger_error` should be silenced though?

Reply:
Yes 👍